<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonWB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>微博</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonZhiHu">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>知乎</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDaily">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>今日头条</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
